### PR TITLE
fix: server crash if "localized" is used as property name

### PR DESCRIPTION
**Pull request by robertwitt**
This PR fixes a server crash at startup if "localized" is used as property name.

```
INTERNAL ERROR] TypeError: Cannot read properties of undefined (reading 'endsWith')
    at _isLocalized (XXX/node_modules/@cap-js/graphql/lib/schema/util/index.js:6:80)
```

### CODE REVIEW

File: lib/schema/util/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const hasScalarFields = entity =>
     ([, el]) => !(shouldElementBeIgnored(el) || el.isAssociation || el.isComposition)
   )
 
-const _isLocalized = element => element.name === 'localized' && element.target.endsWith('.texts')
+const _isLocalized = element => element.name === 'localized' && element.target && element.target.endsWith('.texts')
```
Comment (schwma) [inline lib/schema/util/index.js:6]:
```suggestion
const _isLocalized = element => element.name === 'localized' && element.target?.endsWith('.texts')
```

**Review by schwma — CHANGES_REQUESTED** [review]
Thanks for the PR!

We actually have a similar schema generation test using the model [test/resources/edge-cases/srv/field-named-localized.cds](https://github.com/cap-js/cds-adapter-graphql/blob/e1b7602cba8c865897f6baf183a6d1c36c01365d/test/resources/edge-cases/srv/field-named-localized.cds), but unfortunately it only checked associations and compositions named "localized" and not scalar types which don't have a target.

Could you please move your schema test to the cds model mentioned above?

Could you please also add a changelog entry?

**Comment by robertwitt** [comment]
@schwma I addressed your comments. Please review again.

**Review by schwma — APPROVED** [review]
Great, thanks!

**Comment by robertwitt** [comment]
@schwma Could you merge the PR? I am not authorized.
